joomla/joomla-cms
# Display additional information link in update component

This PR will add a new row to the update information table in the Joomla! Update Component which will display a link for additional information (the `<infourl>` element of the update XML).  For future Joomla! updates, this will be linked directly to the release announcement versus the `joomla.org` homepage as it is now.
### Testing Instructions

1) Apply this patch to a 3.3.6 or earlier Joomla! installation
2) Set the Joomla! Update Component's `Update server` setting to Testing
3) Return to the update component, and if the update to `3.3.7-dev` is not displayed, purge your update cache
4) Verify the table looks like the below screenshot

![screen shot 2014-10-04 at 7 01 11 am](https://cloud.githubusercontent.com/assets/368545/4515316/fa4499f4-4bb5-11e4-88e1-d84c790c0392.png)
